changelog.md

```diff
@@ -387,6 +387,8 @@
 
 - Added `--declaredlocs` to show symbol declaration location in messages.
 
+- You can now enable/disable VM tracing in user code via `vmutils.vmTrace`.
+
```

Comment:
But why would the ordinary Nim user want to do that?

Reply:
i can remove it from changelog, i don't care much; but the feature is useful for anyone that's ever had to debug macros or user code that runs in VM